src/ExposeNodeParser.ts

```diff
@@ -49,4 +53,49 @@ export class ExposeNodeParser implements SubNodeParser {
 
         return argumentIds.length ? `${fullName}<${argumentIds.join(",")}>` : fullName;
     }
+
+    private isFromLib(node: ts.Node): boolean {
+        const sourceFile = node.getSourceFile();
+        if (!sourceFile) {
+            return false;
+        }
+        return /[\\/]typescript[\\/]lib[\\/]/i.test(sourceFile.fileName);
+    }
+
+    private shouldInline(node: ts.Node, type: BaseType, context: Context): boolean {
```

Comment:
How does this handle cases where a symbol is created that can be used in multiple cases? Then it would be beneficial to have the alias, no?

Reply:
Hmm, we have a principle to reuse definitions (even if not explicitly exported). It helps with generating code in other programming languages for example.

I wonder whether we can take a different approach. We could check the generated schema and inline aliases that are generated (rather than inferred from type names) for those cases where the alias is only used once. This could be something that covers the original case you described as well as other cases. 

Thoughts?